### DISCUSSION TITLE
[Question] is there a way to run android/ios in debug mode(physical device)?

### DISCUSSION THREAD

**Discussion by bitmvk** [flet-dev/flet#3674] [Q&A] [answered]
### Question

when writing apps in flutter we use `flutter run` and select the physical device. Is there a way to do the same using flet?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
- https://flet.dev/docs/getting-started/testing-on-ios
- https://flet.dev/docs/getting-started/testing-on-android

**Reply by bitmvk** [reply]
I am sorry for not elaborating,

i am extending flet as explained [here](https://flet.dev/docs/extend/integrating-existing-flutter-packages)
how to test the app in ios/android physical devices with debug mode enabled

**Reply by bitmvk** [reply]
@ndonkoHenri

**Reply by bitmvk** [reply, marked as answer]
I just read what i wrote. 
sorry again 🙏🙏

what i was asking was how to check the logs of the app.

i found out you can use `flutter logs` to get the logs as told [here](https://stackoverflow.com/questions/63617966/how-to-view-flutter-debug-console-output-for-a-release-version-of-the-app)

Thanks for the response
